crates/apps/src/lib/node/ledger/shell/finalize_block.rs

```diff
@@ -652,6 +503,372 @@ where
                 .expect("Error while marking tx hash as redundant");
         }
     }
+
+    // Evaluate the result of a batch. Commit or drop the storage changes,
+    // update stats and event, manage replay protection.
+    fn evaluate_tx_result(
+        &mut self,
+        response: &mut shim::response::FinalizeBlock,
+        dispatch_result: &Result<()>,
+        tx_result: namada::tx::data::TxResult<protocol::Error>,
+        tx_data: TxData,
+        mut tx_logs: TxLogs,
+    ) {
+        // Check the commitment of the fee unshielding regardless of the
```

Comment:
This will change somewhat as part of the MASP fee rework, right?

Reply:
Correct, this logic will be removed